Formula/qt.rb

```diff
@@ -98,7 +92,6 @@ class Qt < Formula
 
   fails_with gcc: "5"
 
-  # fix build with Xcode 13+
   patch :DATA
```

Comment:
The comments you added for the patches should be here. Putting it where you did makes it much easier to miss.